depscan/lib/package_query/cargo_pkg.py

```diff
@@ -112,5 +120,7 @@ def cargo_pkg_risk(pkg_metadata, is_private_pkg, scope, pkg):
         risk_metrics[f"pkg_{scope}_scope_risk"] = True
         risk_metrics[f"pkg_{scope}_scope_value"] = 1
 
+    risk_metrics['pkg_includes_binary_risk'] = does_binary_exist(versions_dict.get(pkg.get("version"), {}))
```

Comment:
We also need a value based on the number of binaries

Reply:
We also handle the cases where we check the latest version if no version number is specified?

Reply:
added both of these changes